codec-compression/src/main/java/io/netty/handler/codec/compression/ZlibCodecFactory.java

```diff
@@ -104,27 +104,42 @@ public static ZlibEncoder newZlibEncoder(int compressionLevel, int windowBits, i
         }
     }
 
+    @Deprecated
     public static ZlibDecoder newZlibDecoder() {
+        return newZlibDecoder(0);
+    }
+
+    public static ZlibDecoder newZlibDecoder(int maxAllocation) {
```

Comment:
Please add javadocs, here and elsewhere, that explain the parameters.

Reply:
Added to every `newZlibDecoder` method